osidb/mixins.py

```diff
@@ -148,68 +149,79 @@ class Meta:
         abstract = True
 
 
+class Alert(models.Model):
```

Comment:
Nitpick: I'd prefer to store the model itself in `osidb/models.py` for consistency with most other models.

Reply:
The only reason I added it here is to avoid a circular dependency: the `AlertMixin` uses `Alert`, and models in the `osidb/models.py` file use `AlertMixin`